requirements.txt

```diff
@@ -4,7 +4,7 @@ pyotp~=2.6.0
 python-magic~=0.4.24
 redis~=3.5.3
 repoze.who~=2.4
-repoze.who-use-beaker~=0.3
+git+https://github.com/mediasuitenz/repoze.who-use_beaker.git@master
```

Comment:
Can this use a specific version instead of a branch?

Reply:
Good call, I'll tag that repo and update here

Reply:
Done :+1: